src/library/mixxxlibraryfeature.cpp

```diff
@@ -56,6 +57,7 @@ MixxxLibraryFeature::MixxxLibraryFeature(Library* pLibrary,
             << "library." + LIBRARYTABLE_REPLAYGAIN
             << "library." + LIBRARYTABLE_FILETYPE
             << "library." + LIBRARYTABLE_DATETIMEADDED
+            << "last_played.datetime_played"
```

Comment:
The table name is defined as a constant, but you don't use it?

Reply:
fixed